prv_candidates_step/step.py

```diff
@@ -76,17 +76,18 @@ def post_execute(self, result: Tuple):
         # Produce a message with the non_detections
         for index, alert in enumerate(result[0]):
             non_detections = result[2][index]
-            self.produce_scribe(non_detections, aid=alert["aid"])
+            self.produce_scribe(non_detections, candid=alert["candid"])
 
         return result
 
-    def produce_scribe(self, non_detections: List[dict], aid: str):
+    def produce_scribe(self, non_detections: List[dict], candid: Union[str, int]):
         for non_detection in non_detections:
             scribe_data = {
                 "collection": "non_detection",
                 "type": "update",
-                "criteria": {"aid": aid},
+                "criteria": {"_id": candid},
```

Comment:
Para tener tipos consistentes en la base de datos

```suggestion
                "criteria": {"_id": str(candid)},
```